package.json

```diff
@@ -12,32 +12,33 @@
     "url": "git+https://github.com/seek-oss/vocab.git"
   },
   "scripts": {
-    "start.direct": "yarn dev && manypkg run @fixtures/direct compile && yarn start-fixture direct",
-    "start.server": "yarn dev && manypkg run @fixtures/server compile && yarn start-fixture server",
-    "start.simple": "yarn dev && manypkg run @fixtures/simple compile && yarn start-fixture simple",
+    "start:direct": "pnpm dev && pnpm --filter @fixtures/direct compile && pnpm start-fixture direct",
+    "start:server": "pnpm dev && pnpm --filter @fixtures/server compile && pnpm start-fixture server",
+    "start:simple": "pnpm dev && pnpm --filter @fixtures/simple compile && pnpm start-fixture simple",
     "build": "preconstruct build",
     "dev": "preconstruct dev",
     "watch": "preconstruct watch",
-    "format": "yarn format.manypkg && yarn format.prettier",
-    "format.manypkg": "manypkg fix",
-    "format.prettier": "prettier --write .",
-    "lint": "yarn lint.eslint && yarn lint.manypkg && yarn lint.prettier && yarn lint.tsc",
-    "lint.eslint": "eslint --ext .tsx,.ts .",
-    "lint.manypkg": "manypkg check",
-    "lint.prettier": "prettier --check .",
-    "lint.tsc": "tsc",
-    "release": "yarn build && yarn copy-readme-to-packages && changeset publish",
+    "format": "pnpm format:manypkg && pnpm format:prettier",
+    "format:manypkg": "manypkg fix",
+    "format:prettier": "prettier --write .",
+    "lint": "pnpm lint:eslint && pnpm lint:manypkg && pnpm lint:prettier && pnpm lint:tsc",
+    "lint:eslint": "eslint --ext .tsx,.ts .",
+    "lint:manypkg": "manypkg check",
+    "lint:prettier": "prettier --check .",
+    "lint:tsc": "tsc",
+    "release": "pnpm build && pnpm copy-readme-to-packages && changeset publish",
     "test": "jest",
     "test:local": "PUPPETEER_EXPERIMENTAL_CHROMIUM_MAC_ARM=1 jest",
     "copy-readme-to-packages": "ts-node scripts/copy-readme-to-packages",
     "start-fixture": "ts-node test-helpers/src/start-fixture",
     "run-server-fixture": "ts-node test-helpers/src/run-server-fixture",
-    "compile-fixtures": "manypkg run @fixtures/direct compile && manypkg run @fixtures/simple compile && manypkg run @fixtures/server compile"
+    "compile-fixtures": "pnpm --filter @fixtures/* compile"
   },
   "workspaces": [
     "packages/*",
     "fixtures/*",
-    "test-helpers"
+    "test-helpers",
+    "tests"
   ],
```

Comment:
This should be removed since you have a `pnpm-workspace.yaml`

Reply:
Good pickup